entity-framework/core/what-is-new/ef-core-2.1.md

```diff
@@ -0,0 +1,71 @@
+---
+title: What is new in EF Core 2.1 - EF Core
+author: divega
+ms.author: divega
+
+ms.date: 2/20/2018
+
+ms.assetid: 585F90A3-4D5A-4DD1-92D8-5243B14E0FEC
+ms.technology: entity-framework-core
+
+uid: core/what-is-new/ef-core-2.1
+---
+
+# New features in EF Core 2.1
++> [!NOTE]  
++> This release is still in preview.
+
+In this release we have fixed more than a hundred product bugs and added numerous small improvements. Here are some of the new major features:
+
+## Lazy loading
+EF Core now contains the necessary building blocks for anyone to write entity classes that can load their navigation properties on demand. We have also created a new package, Microsoft.EntityFrameworkCore.Proxies, that leverages those building blocks to produce lazy loading proxy classes based on minimally modified entity classes. In order to use these lazy loading proxies, you only need navigation properties in your entities to be virtual.
```

Comment:
Technically more is needed than just virtual properties, but I assume this is intentionally simplified?